django_sorcery/db/middleware.py

```diff
@@ -61,6 +61,8 @@ def process_response(self, request, response):
         finally:
             self.return_response(request, response)
 
+        return self.return_response(request, response)
```

Comment:
finally block already calls `return_response` this will compute it twice. you can safely return from `finally` block:

```python
>>> def foo():
  2     try:
  3         raise ValueError
  4     except ValueError:
  5         pass
  6     finally:
  7         return 5
>>> foo()
5
```

Reply:
thanks for fixing this. looks like my refactor broke this.